.circleci/config.yml

```
@@ -56,13 +56,7 @@ aliases:
         name: Install system dependencies
         command: |
           sudo apt-get update
-          sudo apt-get install -y libicu-dev libidn11-dev libprotobuf-dev protobuf-compiler
-          
-          ## TODO: FIX THESE BUSTER DEPENDANCES
-          sudo wget http://ftp.au.debian.org/debian/pool/main/i/icu/libicu57_57.1-6+deb9u3_amd64.deb
-          sudo dpkg -i libicu57_57.1-6+deb9u3_amd64.deb
-          sudo wget http://ftp.au.debian.org/debian/pool/main/p/protobuf/libprotobuf10_3.0.0-9_amd64.deb
-          sudo dpkg -i libprotobuf10_3.0.0-9_amd64.deb
+          sudo apt-get install -y libicu-dev libidn11-dev libprotobuf-dev protobuf-compiler libprotobuf17 libicu63
```

```suggestion
          sudo apt-get install -y libicu-dev libidn11-dev libprotobuf-dev protobuf-compiler
```
I wouldn't hardcode the ABI version as it will break on a bump and dev requires the latest version with a strict dependency anyway.